docs-client/src/containers/MethodPage/index.tsx

```diff
@@ -212,6 +217,15 @@ export default class MethodPage extends React.PureComponent<Props, State> {
                 <Typography variant="body1" paragraph />
                 {this.state.additionalHeadersOpen && (
                   <>
+                    <>
+                      {this.state.exampleHeaders.length > 1 && (
```

Comment:
Just curious. If `exampleHeaders` didn't set to `additionalHeader` at https://github.com/line/armeria/pull/1466/files#diff-f34c1c4217b18c494f39ebed6d5bfab9R494, isn't it necessary to provide this dropdown menu in order that a user can choose an example header?

Reply:
If there's only one example headers, it is set to `EDIT HTTP HEADERS` automatically.
If there are more than one example headers, this dropdown shows up so that a user can choose.

Reply:
I guess it may still be useful to show the drop-down control even if there is only one example. So that a user can reset the additional headers field with the example headers after editing the additional headers field.